Vue InstantSearch/geo-search/package.json

```
@@ -12,7 +12,7 @@
     "instantsearch.css": "7.3.1",
     "vue": "2.6.10",
     "vue-googlemaps": "0.1.2",
-    "vue-instantsearch": "3.8.1"
+    "vue-instantsearch": "^3.8.1"
```

```suggestion
    "vue-instantsearch": "^4.3.3"
```

3.8.1 isn't the latest :) If updated across the major version, you need to check whether the imports still work (they normally should be the same, unless possibly /es replaced by /vue2/es)